src/app/api/analysis/website/scheduled/route.ts

```
@@ -0,0 +1,264 @@
+  let nextRun = new Date();
+  nextRun.setHours(hours, minutes, 0, 0);
+  
+  switch (frequency) {
+    case 'daily':
+      if (nextRun <= now) {
+        nextRun.setDate(nextRun.getDate() + 1);
+      }
+      break;
+      
+    case 'weekly':
+      const targetDay = dayOfWeek ?? 1; // Default to Monday
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)